rust/openvasd/src/scheduling.rs

```diff
@@ -199,7 +199,17 @@ where
                         queued.push(scan_id);
                     }
                     Err(e) => {
-                        tracing::warn!(%scan_id, %e, "unable to start, removing from queue. Verify that scan using the API");
+                        tracing::warn!(%scan_id, %e, "unable to start, removing from queue and set status to failed. Verify that scan using the API");
+                        self.update_status(
+                            &scan_id,
+                            models::Status {
+                                start_time: None,
+                                end_time: None,
```

Comment:
Not sure, if it is better to set the start and end time, as the scan technically did not start at all

Reply:
I think it's fine to leave start and end time empty.